test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -427,3 +427,74 @@ Feature: Bind an application to a service using annotations
                 ready: true
             """
         Then Secret "binding-request-backend-ann" is empty
+
+
+    Scenario: Binding refer service using group version resource
```

Comment:
@olm tag should be added if we are using OLM Operator backend. I guess we can use https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToServiceAnnotations.feature#L13 by applying crd of the service and then the instance without having to rely on olm backend service

Reply:
oops, sorry @olm

Reply:
I mean the tag `@olm` .. :see_no_evil:

Reply:
tag were added.